src/main/java/com/rackspace/salus/monitor_management/services/MonitorManagement.java

```diff
@@ -2326,4 +2326,17 @@ public RenderedMonitorTemplate renderMonitorTemplate(UUID monitorId, String reso
     }
     return renderedMonitorTemplate;
   }
+
+  public ResourceDTO findResourceByTenantIdAndResourceId(String tenantId, String resourceId) {
+    Resource resource = resourceRepository.findByTenantIdAndResourceId(tenantId, resourceId)
+        .orElseGet(() -> null);
```

Comment:
In this case you can just do

```suggestion
        .orElse(null);
```